nuspec/GregClient.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
   <metadata>
     <id>Greg</id>
-    <version>1.0.0</version>
+    <version>2.0.0</version>
```

Comment:
I don't know what this version number is...but the version number should be controlled by this:
https://github.com/DynamoDS/PackageManagerClient/blob/master/src/GregClient/Properties/globalVersionInfo.cs#L6

or was.